lua/telescope/sorters.lua

```diff
@@ -462,6 +462,9 @@ sorters.get_fzy_sorter = function(opts)
   }
 end
 
+-- TODO: Could probably do something nice where we check their conf
+--          and choose their default for this.
+--          But I think `fzy` is good default for now.
```

Comment:
I send you a diff for that in discord some time ago if you still wanna do it :laughing: